### DISCUSSION TITLE
Unknown Artist Unknown Album - overwrites

### DISCUSSION THREAD

**Discussion by Cowboy-Ry** [automatic-ripping-machine/automatic-ripping-machine#948] [Q&A]
Hello community. Loving ARM, finally getting all my old CDs digitized.
A lot of my cds are not getting identified, so they get put into Unknown Artist Unknown Album (UAUA). This is fine for me because I manually tag them after. However, if I do 2 cd's back to back, ARM simply overwrites the existing UAUA with the new data. Is there a way to have them ripped as something like Unknown Artist A Unknown Album A, then UA B UA B etc? Or maybe to automatically rename and move the folder somewhere of my choosing following the rip?
Thanks all!

**Comment by dbruce-ae05** [comment]
This probably doesn't answer your question, but instead of tagging manually you might try Picard.  It is a related project to MusicBrainz.  Picard can scan your music files by the actual music and compare that to a fingerprint and tag your files correctly.

Related, make sure your config file uses MusicBrainz for CDDB (excerpt from config file):
CDDBMETHOD=musicbrainz
